test/test_unicode_js_library.out

```diff
@@ -1,2 +1,2 @@
-Unicode postjs: àČšñéáúÍųåêâăščếệüçλληνικάбългарскиPусскийСрпскиУкраїнська한국어中文普通话(中国大陆)普通话(香港)中文(台灣)粵語(香港)日本語हिन्दीภาษาไทย
 Unicode snowman ☃ says hello! àČšñéáúÍųåêâăščếệüçλληνικάбългарскиPусскийСрпскиУкраїнська한국어中文普通话(中国大陆)普通话(香港)中文(台灣)粵語(香港)日本語हिन्दीภาษาไทย
+Unicode postjs: àČšñéáúÍųåêâăščếệüçλληνικάбългарскиPусскийСрпскиУкраїнська한국어中文普通话(中国大陆)普通话(香港)中文(台灣)粵語(香港)日本語हिन्दीภาษาไทย
```

Comment:
This is the only affected test.   The reason is that when we `await` for the result of `createWasm` it behaves more like sync instantation, where the `run()` call simply runs main rather than waiting for unresolved dependencies.

This means that `postjs` code, which comes after `main` will now run after main like it would do with sync instantiation. 

I think this is a pretty reasonable change.   Basically `main` runs first if the wasm module is ready, and main run later if the wasm module is not ready.